index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>
+            Hello Microverse
```

Comment:
[Required ] you didn't stick exactly with the requested text which is  (Hello Microverse!) add!
- I know this might seems silly but you have to always stick 100% with projects requirements so you have the perfect code to client or the company want